zrml/prediction-markets/src/mock.rs

```diff
@@ -397,6 +409,44 @@ impl ExtBuilder {
         pallet_balances::GenesisConfig::<Runtime> { balances: self.balances }
             .assimilate_storage(&mut t)
             .unwrap();
+        orml_tokens::GenesisConfig::<Runtime> {
+            balances: (0..69)
+                .into_iter()
+                .map(|idx| (idx, CurrencyId::ForeignAsset(100), INITIAL_BALANCE))
+                .collect(),
+        }
+        .assimilate_storage(&mut t)
+        .unwrap();
+        let custom_metadata = CustomMetadata { allow_in_pool: true, ..Default::default() };
```

Comment:
This should also be tagged with `#[cfg(feature = "parachainn")]`. Might be stylistically better to use https://docs.rs/cfg-if/0.1.2/cfg_if/ here.